tests/Aspire.Components.Common.Tests/Aspire.Components.Common.Tests.csproj

```diff
@@ -2,6 +2,10 @@
 
   <PropertyGroup>
     <TargetFramework>$(DefaultTargetFramework)</TargetFramework>
+    <!-- https://learn.microsoft.com/dotnet/core/testing/microsoft-testing-platform-exit-codes -->
+    <!-- Exit code 8 is "zero tests ran" -->
+    <!-- Currently, none of the tests in this project run in CI. All are ignored -->
+    <TestingPlatformCommandLineArguments>$(TestingPlatformCommandLineArguments) --ignore-exit-code 8</TestingPlatformCommandLineArguments>
```

Comment:
@Youssef1313 this doesn't appear to be used in Microsoft.Testing.Platform.targets, which results in the tests to fail
![image](https://github.com/user-attachments/assets/40dfba01-b7da-4eeb-8d2e-879316b11370)

Should this be tucked on to `_TestRunnerArgs`?
https://github.com/dotnet/aspire/blob/fd413eb66b219013ad4b8bc64a459c84a82fa446/eng/Xunit3/Microsoft.Testing.Platform.targets#L23-L31

Reply:
@RussKie `TestingPlatformCommandLineArguments` is currently used when invoking `dotnet test` but not when invoked via Arcade.

Reply:
What would be the right way to fix this?

Reply:
I think you can simply append `TestingPlatformCommandLineArguments` to `_TestRunnerArgs`.

Reply:
Hmm, what I'm suggesting won't probably be possible without my PR on Arcade side. For now I think you could duplicate the `--ignore-exit-code 8` on both sides.

Reply:
Or simply just rename Aspire.Components.Common.Tests to end with TestUtilities if it's really not intended to be a test project, then you don't need the --ignore-exit-code 8 at all.

Reply:
> Or simply just rename Aspire.Components.Common.Tests to end with TestUtilities if it's really not intended to be a test project, then you don't need the --ignore-exit-code 8 at all.

Yep, working on it

Reply:
@RussKie You need to reference `xunit.v3.extensibilty.core` in that project